src/reader_util.cpp

```diff
@@ -207,24 +207,28 @@ std::vector<std::string> ReaderUtil::DetectEncodings(StringView string) {
 }
 
 std::string ReaderUtil::GetEncoding(StringView ini_file) {
+#if LCF_SUPPORT_INI
 	INIReader ini(ToString(ini_file));
 	if (ini.ParseError() != -1) {
 		std::string encoding = ini.Get("EasyRPG", "Encoding", std::string());
 		if (!encoding.empty()) {
 			return ReaderUtil::CodepageToEncoding(atoi(encoding.c_str()));
 		}
 	}
+#endif
```

Comment:
Suggestion: Maybe add some message instead of failing silently?
```c++
#else
Log::Warning("Could not get encoding from ini file, disabled in this liblcf build.");
#endif
```

Reply:
Yeah should probably do that.